qiskit/providers/ibmq/experiment/experiment.py

```diff
@@ -66,6 +69,9 @@ def __init__(
                 group from the provider is used.
             project: The project to which this experiment belongs. If not specified the
                 project from the provider is used.
+            visibility: The visibility of the experiment which determines who can
+                see it when listing experiments.
+            owner: The user ID for the owner of the experiment.
```

Comment:
Same as above. Perhaps a few words that explains this is not expected to be set by the user when creating a new experiment